examples/extend-graphql-schema-nexus/schema.ts

```diff
@@ -28,3 +32,75 @@ export const lists = {
     },
   }),
 };
+
+export function extendGraphqlSchema(baseSchema: GraphQLSchema) {
+  const NexusPostQuery = nexus.extendType({
+    type: 'Query',
+    definition(t) {
+      t.field('nexusPosts', {
+        type: nexus.nonNull(nexus.list('Post')),
+        args: {
+          id: nexus.nonNull(nexus.stringArg()),
+          seconds: nexus.nonNull(nexus.intArg({ default: 600 })),
+        },
+
+        async resolve(root, { id, seconds }, context) {
+          const cutoff = new Date(Date.now() - seconds * 1000);
+
+          // Note we use `context.db.Post` here as we have a return type
+          // of [Post], and this API provides results in the correct format.
+          // If you accidentally use `context.query.Post` here you can expect problems
+          // when accessing the fields in your GraphQL client.
+          return context.db.Post.findMany({
+            where: { author: { id: { equals: id } }, publishDate: { gt: cutoff } },
+          }) as Promise<Lists.Post.Item[]>; // TODO: nexus doesn't like <readonly Post[]>
+        },
+      });
+    },
+  });
+
+  const NexusThing = nexus.objectType({
+    name: 'NexusThing',
+    definition(t) {
+      t.int('id');
+      t.string('title');
+    },
+  });
+
+  const NexusThingQuery = nexus.extendType({
+    type: 'Query',
+    definition(t) {
+      t.nonNull.list.field('things', {
+        type: NexusThing,
+        resolve() {
+          return [
+            { id: 1, title: 'Keystone' },
+            { id: 2, title: 'Prisma' },
+            { id: 3, title: 'Nexus' },
+          ];
+        },
+      });
+    },
+  });
+
+  return nexus.makeSchema({
+    mergeSchema: {
+      schema: baseSchema,
+    },
+    types: {
+      NexusThing,
+      NexusPostQuery,
+      NexusThingQuery,
+    },
+
+    // Typescript output settings, probably something you might commit in dev
+    shouldGenerateArtifacts: process.env.NODE_ENV !== 'production',
```

Comment:
Outputs with `yarn build` only (webpack)
`yarn dev` doesn't work, probably something to do with `esbuild`